src/pages/VestingClaim.tsx

```diff
@@ -0,0 +1,191 @@
+  const [remainingAmount, setRemainingAmount] = useState("0.0")
+  const [vestingContract, setVestingContract] = useState<Vesting>()
+
+  useEffect(() => {
+    const fetchBeneficiaries = async () => {
+      if (!library || !chainId) return
+      const provider = new ethers.providers.Web3Provider(
+        window.ethereum as never,
+      )
+      const SDLContract = new EthersContract(
+        SDL_TOKEN_ADDRESSES[chainId],
+        SDL_TOKEN_ABI,
+        provider.getSigner(),
+      ) as Sdl
+      const vestingContractDeployedFilter = SDLContract.filters.VestingContractDeployed(
+        null,
+        null,
+      )
+      try {
+        const events = await SDLContract.queryFilter(
+          vestingContractDeployedFilter,
+        )
+        const currentBeneficiary = events.filter(
+          (event) => event.args?.beneficiary === account,
+        )
+        setIsValidBene(currentBeneficiary.length > 0)
+        if (isValidBene) {
+          const vestingContract = new EthersContract(
+            currentBeneficiary[0].args?.vestingContract,
+            INVESTOR_EMPLOYEE_VESTING_CONTRACT_ABI,
+            provider.getSigner(),
+          ) as Vesting
+          setVestingContract(vestingContract)
+          try {
+            const remainingAmount = commify(
+              formatBNToString(
+                await SDLContract.balanceOf(vestingContract.address),
+                18,
+                2,
+              ),
+            )
+            setRemainingAmount(remainingAmount)
+          } catch (err) {
+            console.error(err)
+            notifyCustomError({
+              ...(err as Error),
+              message: "Unable to get total pending amount",
+            })
+          }
+          try {
+            const claimableVestedAmount = commify(
+              formatBNToString(await vestingContract.vestedAmount(), 18, 2),
+            )
+            setClaimableVestedAmount(claimableVestedAmount)
+          } catch (err) {
+            console.error(err)
+            notifyCustomError({
+              ...(err as Error),
+              message: "Unable to get vested amount",
+            })
+          }
+        }
+      } catch (err) {
+        console.error(err)
+        notifyCustomError({
+          ...(err as Error),
+          message: "Unable to query for vesting contracts",
+        })
+      }
+    }
+    void fetchBeneficiaries()
+  }, [account, chainId, library, isValidBene])
+
+  const onClaimClick: () => void = async () => {
+    if (!vestingContract) return
+    try {
+      const txn = await vestingContract.release()
+      notifyHandler(txn?.hash, "claim")
+      await txn?.wait()
+      setClaimableVestedAmount("0.0")
+    } catch (err) {
+      console.error(err)
+      notifyCustomError({
+        ...(err as Error),
+        message: "Unable to claim vested tokens",
+      })
+    }
+  }
+
+  return (
+    <>
+      <TopMenu activeTab={t("risk")} />
```

Comment:
On this page, does we need to active risk?

Reply:
I just changed this to n/a to swallow that logic.